src/ui/components/Library/Team/View/FilterBar.tsx

```diff
@@ -1,38 +0,0 @@
-import React, { ChangeEvent, KeyboardEvent, useContext } from "react";
-
-import { TextInput } from "../../../shared/Forms";
-import { FilterContext } from "./FilterContext";
-import { FilterDropdown } from "./FilterDropdown";
-import { ViewContext } from "./ViewContextRoot";
-
-export function FilterBar() {
-  const { displayedString, setDisplayedText, setAppliedText } = useContext(FilterContext);
-  const { view } = useContext(ViewContext);
-
-  const onChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setDisplayedText(e.target.value);
-  };
-  const onKeyPress = (e: KeyboardEvent<HTMLInputElement>) => {
-    if (e.key === "Enter") {
-      setAppliedText(e.currentTarget.value);
-    }
-  };
-
-  const showFilterInput = view === "recordings";
```

Comment:
This check wasn't doing anything anyway. The parent component already bailed out if we weren't in the "recordings" view.